### ISSUE TITLE
Audio Error

### ISSUE THREAD

**Issue by alice101-dev** [Livruen/Icinga2-Dashboard-with-sound#4]
Host not Down but audio always repeat, "Hosted Down"

**Comment by Livruen** [comment]
Thank you for your question,
if you want to stop the sound "Host is down" please click “Acknowledge Unhandled Problem Hosts”.
The sound will annoying you as long as you acknowledge the problem ;)

**Comment by Livruen** [comment]
If the List has "unacknowledged problems" send a signal "Host is down"
Please check if all hosts problems are acknowledged.

**Comment by alice101-dev** [comment]
thanks for expert and for answer my question,

**Comment by Livruen** [comment]
my pleasure :)

**Comment by thachnv92** [comment]
Hi Ms @Livruen 
I enabled module "imi", but audio always repeat "Host Down" also all host is UP.
I think audio only play when have one or more host is DOWN.

Please help me answer my question.
Thanks ^^

**Comment by thachnv92** [comment]
Hello Mr.@abdidarmawan007
Could you share me your solution for this issue?
I want to browser play sound alarm when critial/warning/host down occur.

Thanks.

**Comment by Livruen** [comment]
Please check if all hosts problems are **acknowledged**.If they are UP but not acknowledged then still there will be an audio output 'Host is down'

**Comment by thachnv92** [comment]
Sorry, I don't understand "**acknowledged**"?
Because we always have a few services CRITICAL because memory full.
But **Down hosts 0**

**Comment by Livruen** [comment]
If i remember correctly (Sorry I worked with Icinga 2 years ago), there should be a possibility to acknowledge the problem handling. Maybe this link could help https://www.icinga.com/docs/icinga2/latest/doc/08-advanced-topics/
If there is a Host which is not acknowledged and UP then it will 'blink'.

**Comment by Livruen** [comment]
The same with the CRITICAL. I it is not acknowledged it will blink. For more details check the code in imi/application/views/scripts/critical/index.phtml

**Comment by thachnv92** [comment]
Yeap. Thank Ms @Livruen 
Sorry if I bother you.

**Comment by cces40115** [comment]
Hi Livruen,
i use your imi module is working, it's so awesome.
but i want to ask you, imi module can be add to dashboard?
like the other icingaweb function have Url can add to dashboard. ex: monitoring/list/hosts
i want let it show the remote on current incidents this dashboard. but i tried to find the Url by imi module is failed.
Do you have any suggestions?

**Comment by cces40115** [comment]
Hi @Livruen ,
i try to add the dashlet: imi/warnings to dashboard let it can stay on the main dashboard help me monitor.
it's work and can see the warning status. but the warning sound is disable on the dashboard.
Do you have any suggestions?
thanks.

**Comment by Livruen** [comment]
Hi @cces40115 ,
Did you try this? 
Linux:
Add the module into /usr/share/icingaweb2/modules
Go into usr/share/icingaweb2/public and make a symbolik link

ln -s ../modules/imi/public/sound sound

Maybe you acknowledged the warnings and thtas why the sound is not comming. Is it blinking?

**Comment by cces40115** [comment]
Yes, i already followed this step, actually the sound alert is working on the imi's Warnings page. 
i mean my method is try to add the dashlet: imi\warning to dashboard let imi's Warning page can 
detect service problem and auto play the sound alert when i stay the dashboard page.
this is my lab chart: https://imgur.com/7rQ7l5J

this method look like not feasible, i try to made it like nagios or zabbix trigger.
but thanks your imi module, it's pretty good to identify the alert syslog. :)

**Comment by Livruen** [comment]
Maybe the problem is the path to the sound.  The sounds are now in the workspace of imi. Try to put the sounds into the Dashboard folder and set the proper path.
This is only a guess ...  sorry for the poor help.

I am happy that the plugin helps some people :)
